README.md

```diff
@@ -48,10 +48,10 @@ is updated.
 
 ## Custom Cops
 1. [RailsConfiguration](https://github.com/ezcater/ezcater_rubocop/blob/master/lib/rubocop/cop/ezcater/rails_configuration.rb) - Enforce use of `Rails.configuration` instead of `Rails.application.config`.
+1. [RspecRequireGqlErrorHelpers](https://github.com/ezcater/ezcater_rubocop/blob/master/lib/rubocop/cop/ezcater/require_gql_error_helpers.rb) - Use the helpers provided by `GQLErrors` instead of raising `GraphQL::ExecutionError` directly.
```

Comment:
Think you might've forgotten to kill the `Rspec` here

Reply:
Bahhhhhhh. Thanks!